client/electron/snap/local/desktop-launch.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+# Capture the current enviroment for the future xdg-open calls.
+XDG_OPEN_ENVIRONMENT=$(export; echo 'xdg-open "$@"')
```

Comment:
`export` is provided by the shell (here, `bash` so that it produces a valid bash shell script to execute). That  means a correct handling of spaces and newlines in the environment variables, which `env` does not do properly.